src/Driver/SQLSrv/Driver.php

```diff
@@ -54,6 +54,14 @@ public function connect(
             $driverOptions['PWD'] = $params['password'];
         }
 
+        if (isset($params['Encrypt'])) {
```

Comment:
As these parameters are specific for this driver, I think you should use the `driverOptions` parameter instead (see [Custom driver options](https://www.doctrine-project.org/projects/doctrine-dbal/en/current/reference/configuration.html#custom-driver-options)).